libs/language-server/src/lib/builtin-library/stdlib.ts

```diff
@@ -3,14 +3,14 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 
 import { IOType, type PrimitiveValueType } from '../ast';
-import { PrimitiveValuetypes } from '../ast/wrappers/value-type/primitive/primitive-value-types';
+import { PrimitiveValueTypeProvider } from '../ast/wrappers/value-type/primitive/primitive-value-type-provider';
 
 import { PartialStdLib } from './generated/partial-stdlib';
 
 export function getBuiltinValuetypesLib() {
-  const primitiveValuetypes = Object.values(PrimitiveValuetypes).map(
-    parseBuiltinValuetypeToJayvee,
-  );
+  const primitiveValuetypes =
+    new PrimitiveValueTypeProvider().Primitives.getAll() // instantiation is okay here as it has no side effects: it is only parsed to string
```

Comment:
TODO for future PRs: read builtin value types from jv files instead creating from code.

Reply:
Extracted to issue #551